src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,51 @@
+import { JsonToCol, LanguagesInput } from "../../@types/interfaces";
+
+const mapLangToSheetColumns = (languages: LanguagesInput) => {
+  let columnTitle: Array<object> = [];
+  Object.keys(languages).map(
+    (title: string) =>
+      (columnTitle = [
+        ...columnTitle,
+        { type: "text", title: `${title}`, width: 150 },
+      ])
+  );
+  return columnTitle;
+};
+
+const setKeys = (languages: LanguagesInput): Array<string> => {
+  let languagesKeys: Array<string> = [];
+  Object.keys(languages).map(
+    (titles) => (languagesKeys = [...languagesKeys, titles])
+  );
+  const keyValue = Object.keys(languages[languagesKeys[0]]);
+  return keyValue;
+};
+
+const mapLangToSheetData = (languages: LanguagesInput) => {
```

Comment:
Please come up with simplification of this function. I repeat previous review comments:
* do not use any type (Array<object> is basicaly array of any types)
* do not prepare all the variables at the beginning

Reply:
think of simpler implementation based on: map, and sort at the end
think of it as matrix transpose

Reply:
Are you planning to work on this review comment?

Reply:
1. do not use any type (Array is basically an array of any types)
2. do not prepare all the variables at the beginning
3. simplification of this function

which one?

Reply:
1. all type removed (only have the interface )
2. Assigned all variable 
3. Simplified 2 functions into 1

Reply:
Which one?